### ISSUE TITLE
Summoners War

### ISSUE THREAD

**Issue by sddevstudio** [glasklart/hd#6346]
**App Name:** Summoners War
**Bundle ID:** com.com2us.smon.normal.freefull.apple.kr.ios.universal
**iTunes ID:** <a target="_blank" href="http://getart.dreamnet.at?id=852912420">852912420</a>
**iTunes URL:** <a target="_blank" href="https://itunes.apple.com/us/app/summoners-war/id852912420?mt=8&uo=4">https://itunes.apple.com/us/app/summoners-war/id852912420?mt=8&uo=4</a>
**App Version:** 1.7.5
**Seller:** Com2uS Corp.
**Developer:** <a target="_blank" href=https://itunes.apple.com/us/developer/com2us-corp./id299175870?uo=4>© Com2uS Corp.</a>
**Supported Devices:** iPad2Wifi, iPad23G, iPhone4S, iPadThirdGen, iPadThirdGen4G, iPhone5, iPodTouchFifthGen, iPadFourthGen, iPadFourthGen4G, iPadMini, iPadMini4G, iPhone5c, iPhone5s, iPhone6, iPhone6Plus, iPodTouchSixthGen
**Original Artwork:**
<img src="http://is1.mzstatic.com/image/thumb/Purple3/v4/5e/6f/7e/5e6f7ed9-c2e6-064d-6bcb-ab3b42459871/mzl.teslvzhs.png/0x0ss-85.jpg" width="150" height="150" />
**Accepted Artwork:**
![2x](http://preview.dreamnet.at?image=https://cloud.githubusercontent.com/assets/2068130/9975237/1078d588-5eb5-11e5-83b6-a0b304ee4b0a.png)

**Comment by sddevstudio** [comment]
```
  <key>CFBundleIcons</key>
  <dict>
    <key>CFBundlePrimaryIcon</key>
    <dict>
      <key>CFBundleIconFiles</key>
      <array>
        <string>AppIcon29x29</string>
        <string>AppIcon40x40</string>
        <string>AppIcon57x57</string>
        <string>AppIcon60x60</string>
      </array>
      <key>UIPrerenderedIcon</key>
      <true/>
    </dict>
  </dict>
  <key>CFBundleIcons~ipad</key>
  <dict>
    <key>CFBundlePrimaryIcon</key>
    <dict>
      <key>CFBundleIconFiles</key>
      <array>
        <string>AppIcon29x29</string>
        <string>AppIcon40x40</string>
        <string>AppIcon57x57</string>
        <string>AppIcon60x60</string>
        <string>AppIcon50x50</string>
        <string>AppIcon72x72</string>
        <string>AppIcon76x76</string>
      </array>
      <key>UIPrerenderedIcon</key>
      <true/>
    </dict>
  </dict>

**Comment by dreamnet** [comment]
@sddevstudio Thx for the extra info, but we are using the iconbundles method, so we don't need it.

**Comment by sddevstudio** [comment]
@dreamnet sounds good, I have been out of the loop for awhile, trying to get back in the groove

**Comment by dreamnet** [comment]
Can't do the original....
![2x](http://preview.dreamnet.at?image=https://cloud.githubusercontent.com/assets/2068130/9975227/9bc9b248-5eb4-11e5-8fbf-4067f429e594.png)
https://cloud.githubusercontent.com/assets/2068130/9975227/9bc9b248-5eb4-11e5-8fbf-4067f429e594.png
--- ---
Source:
https://cloud.githubusercontent.com/assets/2068130/9975228/aa052176-5eb4-11e5-8b0d-f70360069717.png

**Comment by sddevstudio** [comment]
Looks great thanks! 

I have tried to manually add this with the standard naming: com.com2us.smon.normal.freefull.apple.kr.ios.universal@2x.png but doesn't appear to be working. Using Anemone I simply hit Apply -> Apply. This worked fine for an existing icon I had changed to see if something was caching. For new icons that haven't been themed yet do you still need to delete the icon cache? or should re-applying theme be good enough?

**Comment by dreamnet** [comment]
@sddevstudio For manually adding this icon navigate to `Library/themes/Glasklart Icons.theme/IconBundles` and copy the icon in this folder using `com.com2us.smon.normal.freefull.apple.kr.ios.universal@2x.png` as file name. Then you normally only need to hit Apply -> Apply in Anemone and the icon should be themed. Anemone automatic clears the cache. If you think you have copyed the image in the right place with the right filename you can use `iCleaner` or `Respring+` to clear the caches.

**Comment by ivanov007** [comment]
I've tried this and iCleaner did the trick, maybe the name is tooooo long :wink: I had some issues with iFile too and I guess it was all because of the long name

**Comment by dreamnet** [comment]
@ivanov007 THX for the info!!! :+1:

**Comment by sddevstudio** [comment]
I couldn't get it working even with iCleaner in the normal theme folder. I created a new shorter named folder: GMI.theme. That seemed to do the trick.

**Comment by ivanov007** [comment]
@sddevstudio mhhh you shouldn't be using folders just rename the file to '<bundle name>@2x.png' and move it for the already existent 'Library/themes/Glasklart Icons.theme/IconBundles' folder

**Comment by sddevstudio** [comment]
@ivan007 that's what I had tried originally and I was thinking it might have been an issue with path length so I created a simple theme folder. I will try and do a little more troubleshooting this evening to see whats going on with my device.

**Comment by ivanov007** [comment]
@sddevstudio check for spaces at the beginning or end of the file name, also the capitalization, for this one all the letters are lowercase, this drove me crazy a few years ago

**Comment by dreamnet** [comment]
@sddevstudio I downloaded the app and copyed the icon manually as I posted above and just did a respring with Apply -> Apply in Anemone and the icon was themed without any problems... iPhone 6 with iOS 8.3.

**Comment by sddevstudio** [comment]
I'm using an iPhone 5 on 8.4, something is just weird with my device I think, because it works just find if I have this on in a completely different theme folder, but even if I move it over it still doesn't work. I am moving the files from my mac to the phone via ssh. Might be something wonky thing that who knows. it is working though, I appreciate your help with this both of you.

**Comment by dreamnet** [comment]
@sddevstudio OK... strange. :confused:  I am addin the icon to the master and close this issue now - the icon will be included in release 4.26 (4.25 is already on it's way to Cydia).
